### DISCUSSION TITLE
Welcome to dnotebook-react Discussions!

### DISCUSSION THREAD

**Discussion by risenW** [opensource9ja/dnotebook-react#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by risenW** [comment]
@AceTheCreator and @steveoni I merged the latest PR from @AceTheCreator. Going forward, we should create issues for sections we'll be working on, so that we can collaborate on this and go as fast as possible. 

So @AceTheCreator if you working on a session, create an issue for it and let us know so that we won't duplicate work. 

Over the weekend I'll create some issue and start work on my end.

**Reply by AceTheCreator** [reply]
@risenW  I think that's a great idea, I'll create issues for sections I'm currently working on and for the ones anybody can choose to work on.

**Comment by AceTheCreator** [comment]
@risenW @steveoni  When I download a notebook, not all the cells are available when i later reupload them for reuse. I've been trying to figure this out for a while but no result yet. Any idea what i can do?

**Reply by steveoni** [reply]
Have you solved this? have you check if the state cell is being properly updated?

**Reply by AceTheCreator** [reply]
@steveoni Thanks I was able to solve it. It has something to do with the DOM

**Comment by AceTheCreator** [comment]
@risenW and @steveoni I've submitted a PR please kindly review it (you might wanna test it locally) and please your feedback will be highly appreciated. The only thing remaining currently is writing tests, which I'll start doing very soon(I've started already tho) @risenW what's the status on GitHub actions?

**Reply by risenW** [reply]
> @risenW and @steveoni I've submitted a PR please kindly review it (you might wanna test it locally) and please your feedback will be highly appreciated. The only thing remaining currently is writing tests, which I'll start doing very soon(I've started already tho) @risenW what's the status on GitHub actions?

Well done and thanks @AceTheCreator I'll set it up later this week.👍🏽
